cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+\subsubsection{Convection using a pressure--temperature look-up table and the rheology of Steinberger and Calderwood (2006)}
+\label{sec:cookbooks-steinberger}
+
+\textit{This section was contributed by Juliane Dannberg and Ren{\'e} Gassm{\"o}ller.}
+
+In this cookbook we will go one step further from the last one and set up a fully compressible mantle convection model using the projected density approximation (where the density is interpolated onto the finite element grid to compute the density gradients in the mass conservtaion equation rather than approximating these gradients using a reference profile or tempertaure/pressure derivatives of the density, see \cite{gassmoller2020formulations}). To compute the material properties, we read in a look-up table of material properties in dependence of temperature and pressure computed using a mineral physics software (in this case, Perple\_X, \cite{connolly2005computation}). The table is based on the thermodynamic database by \cite{stixrude2011thermodynamics} and a pyrolitic composition \cite{ringwood1988nature}. Compared to a 1D profile, a temperature--pressure look-up table has the advantage that material properties are accurate not only around one reference adiabat, but also for strongly deviating pressures and temperatures. This is particularly important at phase transitions, because their depth depends on the temperature and pressure. 
```

Comment:
```suggestion
In this cookbook we will go one step further from the last one and set up a fully compressible mantle convection model using the projected density approximation (where the density is interpolated onto the finite element grid to compute the density gradients in the mass conservation equation rather than approximating these gradients using a reference profile or temperature/pressure derivatives of the density, see \cite{gassmoller2020formulations}). To compute the material properties, we read in a look-up table of material properties in dependence of temperature and pressure, originally computed using a mineral physics software (in this case, Perple\_X, \cite{connolly2005computation}). The table is based on the thermodynamic database of \cite{stixrude2011thermodynamics} and a pyrolitic composition \cite{ringwood1988nature}. Compared to a 1D profile, a temperature-pressure look-up table has the advantage that material properties are accurate not only around one reference adiabat, but also for strongly deviating pressures and temperatures. This is particularly important at phase transitions, because their depth depends on the temperature and pressure. 
```